calico_upgrade/pkg/constants/help.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
+
+const (
+	DatastoreIntro = `Set the Calico datastore access information in the environment variables or
+or supply details in a config file. If you are using config files, use the v1
+format of the config file (--apiconfigv1 param), and the v3 format of the
+config (--apiconfigv3 param). If you are using environment variables, the v1
+environments are all prefixed with "APIV1_"
+
+`
+	Exiting = "Exiting..."
+
+	ReportHelp = `    ` + FileConvertedNames + `
+      This contains a mapping between the v1 resource name and the v3 resource
+      name. This will contain an entry for every v1 resource that was
+      migrated.
+
+    ` + FileNameClashes + `
+      This contains a list of resources that after conversion to v3 have
+      names that are identical to other converted resources. This may occur
+      because name formats in Calico v3 are in some cases more restrictive
+      than previous versions. When the migration code modifies a name an
+      additional short qualifying suffix is appended. In rare circumstances
+      the converted name (with the additional qualifier) may clash with other
+      resources of the same kind.
+
+This may occur
```

Comment:
is there an effect to this short line without any leading spaces on the actual output?

Reply:
Hmm odd.  Yes, the effect is it'll look awful.